src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/EditPreferencesPage.java

```diff
@@ -22,7 +20,7 @@ private EditPreferencesPage openEditingSection(String wikiURL) {
   }
 
   public EditPreferencesPage openEditingSection() {
-    return openEditingSection(urlBuilder.getUrlForWiki(Configuration.getWikiName()));
+    return openEditingSection(urlBuilder.getUrlForWiki());
   }
 
   private EditPreferencesPage openEmailSection(String wikiURL) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openEmailSection" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)